requirements.yml

```diff
@@ -1,107 +1,24 @@
 ---
-
-- name: ome.certbot
-  src: https://github.com/ome/ansible-role-certbot/archive/0.1.0.tar.gz
-  version: 0.1.0
-
-- src: ome.cli_utils
-  version: 1.1.1
-
-- src: ome.deploy_archive
-  version: 0.1.4
-
-- src: ome.docker
-  version: 3.1.1
-
-- src: ome.ice
-  version: 4.3.0
-
-- src: ome.java
-  version: 2.1.0
-
-- name: ome.iptables_raw
-  version: 0.3.1
-
-- src: ome.lvm_partition
-  version: 1.1.1
-
-- name: ome.network
-  version: 1.1.4
-
-- src: ome.nginx
-  version: 2.1.1
-
-- name: ome.nginx_proxy
-  version: 1.15.2
-
-- src: ome.nfs_mount
-  version: 1.3.0
-
 - src: ome.omero_common
-  version: 0.3.4
-
 - src: ome.basedeps
-  version: 1.2.0
-
-- name: ome.omero_prometheus_exporter
-  version: 0.3.6
-
-- name: ome.omero_server
-  version: 4.0.2
-
-- src: ome.omero_user
-  version: 0.3.0
-
-- name: ome.omero_web
-  version: 4.0.1
-
+- src: ome.java
```

Comment:
Are these expected to be examples or the actual playbooks used for deploying the production services?
In the latter case, I think you might want to keep the version rather than installing the latest role by default

Reply:
Pin the versions of roles:

The drawback of this is that the versions will become stale (that happened already).
The advantage of this - I am not sure, maybe we should discuss this ? I thought our roles are pretty well maintained and we do not want to introduce anything to them which is not compliant with our prod playbooks deployment ?

Reply:
> The drawback of this is that the versions will become stale (that happened already).

That's correct, if these are stored, these versions need to be maintained, that's not different from any other dependency.

>  The advantage of this - I am not sure, maybe we should discuss this ?

Primarily a reproducibility argument. If a new version of the role with backwards-incompatible changes is made, this approach means installing the roles and running the playbook from the exact same commit would lead to different outcome.

Reply:
fixed in https://github.com/ome/prod-playbooks/pull/379/commits/25533d4a3fc1fe6f4ab84a177e6875253addef50